### DISCUSSION TITLE
Install dependencies unresovable

### DISCUSSION THREAD

**Discussion by OS4** [albilu/netbeansPython#82] [Q&A] [answered]
When trying to install the Python module I get : 

"Some plugins require plugin Navigate To Test to be installed.
The plugin Navigate To Test is requested in implementation version 19-ba85468772292fd64f188f4022f9d49f77f00b89.  The following plugin is affected:       Python 

Some plugins require plugin Code Coverage Support to be installed.
The plugin Code Coverage Support is requested in implementation version 19-ba85468772292fd64f188f4022f9d49f77f00b89.  The following plugin is affected:       Python 

Some plugins require plugin Common Test Runner UI to be installed.
The plugin Common Test Runner UI is requested in implementation version 19-ba85468772292fd64f188f4022f9d49f77f00b89.  The following plugin is affected:       Python 

Some plugins not installed to avoid potential installation problems."

I cannot find plugins called Navigate To Test, Common Test Runner UI nor Code Coverage Support. Code coverage is available, but it appears to be built in. Are these plugins called something else less obvious? 

Can anyone help?

**Comment by albilu** [comment]
Which version of netbeans are you using?

**Comment by OS4** [comment]
Nightly builds

**Reply by albilu** [reply, marked as answer]
This plugin is designed for official Netbeans releases please use the releases available
